SkiaChart/Charts/ChartBase.cs

```diff
@@ -207,16 +207,35 @@ public abstract class ChartBase : IChart {
             IsAntialias = true,
             Style = SKPaintStyle.Stroke,
             Color = SKColors.Green,
-            StrokeWidth = 5,
+			//StrokeWidth = 5,
```

Comment:
I think we should completely remove the StrokeWidth instead of just commenting it out.